doc/sphinx/yaml/general.rst

```diff
@@ -0,0 +1,92 @@
+.. highlight:: yaml
+
+*****************
+General Structure
+*****************
+
+Sections
+--------
+
+The top level of a Cantera YAML input file is a mapping which defines different
+input file sections. Each section consists of a list of mappings which define
+objects of the same type, e.g. reactions, species, phases, or elements. The
+``phases`` section of an input file contains all of the phase definitions.
+Multiple sections containing reaction, species, or element definitions can be
+used. The specific names ``reactions``, ``species``, and ``elements`` are used
+as defaults when looking for reactions, species, and elements to add to a phase.
+A simple input file has the following structure::
+
+    phases:
+    - name: gas
+      thermo: ideal-gas
+      additional: fields
+    - name: solid
+      thermo: model-name
+      additional: fields
+
+    species:
+    - name: A
+      additional: fields
+    - name: B
+      additional: fields
+    - name: C
+      additional: fields
+
+    reactions:
+    - equation: A + B <=> C + D
+      additional: fields
+    - equation: A + C <=> 2 D
+      additional: fields
+
+Units
+-----
+
+While Cantera generally works internally in SI units, input values can be
+provided using a number of different units.
+
+Compound units are written using the asterisk (``*``) to indicate
+multiplication, the forward slash (``/``) to indicate division, and the caret
+(``^``) to indicate exponentiation. Exponents can include negative and decimal
+values. Standard one-letter metric prefixes can be applied to any unit.
+Supported base units are:
+
+- Mass: ``g``
+- Length: ``m``, ``micron``, ``angstrom``, ``Å``
+- Time: ``s``, ``min``, ``hr``
+- Temperature: ``K``, ``C``
+- Current: ``A``
+- Quantity: ``mol`` (gram mole), ``gmol``, ``mole``, ``kmol``, ``kgmol``, ``molec``
+
+Supported compound units are:
+
+- Energy: ``J``, ``cal``, ``erg``, ``eV``
+- Activation Energy: ``J/kmol``, ``K``
```

Comment:
Should `cal/mol` be here as well? Do we need to include all of Energy, Activation Energy, and Quantity? I guess this is probably for backwards compatibility, but does anyone actually use all three? Are there sets of fields where it makes sense to include all three?

Reply:
`cal/mol` and any other unit that is energy / quantity will automatically work. I didn't think it was necessary to list all the possibilities, but maybe it would be worth writing "J/kmol, cal/mol, etc., K" here (to clarify that K is the odd one out)?

You need to be able to set activation energy explicitly if you want to use kelvin, and that doesn't imply a value for either the energy or quantity units, so I think all three need to be available.